### PR TITLE
Keycloak

### PR DESCRIPTION
**Pull request by lrohner**

### CODE REVIEW

File: file-storage/src/main/resources/application.yml [hunk 1]
```diff
@@ -11,28 +11,29 @@ management:
     web:
       exposure:
         include: "*"
+keycloak:
```
Comment (egrozdani) [inline file-storage/src/main/resources/application.yml:14]: Is this this the same configuration prod profile also?

File: file-storage/src/test/java/ch/valtech/kubernetes/microservice/cluster/filestorage/web/rest/FileStorageControllerIt.java [hunk 2]
```diff
@@ -69,11 +62,11 @@ void shouldFailUploadNewFile() {
...
         FILENAME,
         MediaType.TEXT_PLAIN_VALUE,
-        "test content".getBytes()
+        new byte[0]
     );
 
-    mockMvc.perform(multipart("/api/files").file(file).with(getBearerToken(USER)))
-        .andExpect(status().isForbidden());
+    mockMvc.perform(multipart("/api/files").file(file))
+        .andExpect(status().isBadRequest());
```
Comment (egrozdani) [inline file-storage/src/test/java/ch/valtech/kubernetes/microservice/cluster/filestorage/web/rest/FileStorageControllerIt.java:69]: this test was testing a forbidden error when a somebody with ROLE_USER is authenticated. Is there any way to achieve that with Keycloak?

File: file-storage/src/main/resources/application.yml (hunk 1 above)
Comment (lrohner) [reply file-storage/src/main/resources/application.yml:14]: yes

File: file-storage/src/test/java/ch/valtech/kubernetes/microservice/cluster/filestorage/web/rest/FileStorageControllerIt.java (hunk 2 above)
Comment (lrohner) [reply file-storage/src/test/java/ch/valtech/kubernetes/microservice/cluster/filestorage/web/rest/FileStorageControllerIt.java:69]: I disabled security for testing. I think there is but might be a bit complex

**Pull request merged**